sdk/identity/identity/package.json

```diff
@@ -82,11 +82,11 @@
     "@azure/core-http": "^1.1.6",
     "@azure/core-tracing": "1.0.0-preview.9",
     "@azure/logger": "^1.0.0",
-    "@azure/msal-node": "^1.0.0-alpha.8",
+    "@azure/msal-node": "1.0.0-alpha.8",
     "@opentelemetry/api": "^0.10.2",
     "events": "^3.0.0",
     "jws": "^4.0.0",
-    "msal": "^1.0.2",
+    "msal": "1.4.0",
```

Comment:
I've got a PR here: https://github.com/Azure/azure-sdk-for-js/pull/11871 that is updating the msal stuff. I'd skip this for now. We'll be doing more MSAL updates in future release, too.

Reply:
Reverted the changes to `identity`, will wait for your PR to merge first